web-client/src/components/figma/BlockStyles.tsx

```diff
@@ -95,6 +96,22 @@ const SettingsListWrapper = styled.div`
   padding: 10px;
   background-color: ${COLORS.greyBackground};
   height: 100vh;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+`;
+
+const SettingsListItemWrapper = styled.div`
+  margin: -25px 0px;
```

Comment:
Hey @Tapudp , these are hardcoded margin values which means that when I add another item to the list, it breaks and appears on top of the previous items.

Originally this was named as `SettingsListNameWrapper ` and I assume you were going to create another wrapper for another new item, but this isn't the approach to be taken for such UI Components. 

I see that you have used flexbox in the `SettingsListWrapper` and that is the correct direction to proceed however, you need to make use of that here for each item so that we'd have to just use one item wrapper CSS and the items would automatically get stacked one under the other like I asked for in the previous review